satellite/satellitedb/migrate.go

```diff
@@ -362,6 +362,13 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					)`,
 				},
 			},
+			{
+				Description: "Initiate Network reset",
+				Version:     9,
+				Action: migrate.SQL{
+					`DELETE FROM PATHDATA`,
```

Comment:
is this the only table we want to clear? okay

Reply:
If all pointers are gone, we should be fine afaik.
All other services are depending on them and i didnt want to remove any billing information.

Main issue is, that i cannot run that query on the unit tests, as they arent aware of the table..